src/components/SingletonEditor.tsx

```diff
@@ -30,53 +31,97 @@ interface UseSingletonEditorConfig {
   onSave: (content: Content) => void;
 }
 
+interface UnlayerEditor extends EmailEditor {
+  removeEventListener: any;
```

Comment:
What do you think about using a more realistic type in place of any?

Reply:
💪🏻